docs/user-guide/generate-certificates.md

```diff
@@ -14,32 +40,42 @@ zowe:
   setup:
     certificate:
       type: PKCS12
-      dname:
-        caCommonName: 
-        commonName:
-        orgUnit:
-        org:
-        locality:
-        state:
-        country:
-      validity: 3650
-      pkcs12:
-        directory: /global/zowe/keystore
+        directory: /var/zowe/keystore
+        lock: true
         name: localhost
         password: password
-      caAlias: local_ca
+        caAlias: local_ca
         caPassword: local_ca_password
-        import:
-          keystore:
-          password:
-          alias:
+      dname:
+        caCommonName: ""
+        commonName: ""
+        orgUnit: ""
+        org: ""
+        locality: ""
+        state: ""
+        country: ""
+      validity: 3650
+      san:
+        - dvipa.my-company.com
+        - 12.34.56.78
 ```
 
-To assist with updating `zowe.yaml` the values to generate a self signed PKCS12 certificate are included in the section beginning ` # >>>> Certificate setup scenario 1`.  Other certificate scenarios lower down in the `zowe.yaml` file are commented out.
+To assist with updating `zowe.yaml`, the values to generate a self-signed PKCS12 certificate are included in the section beginning with`# >>>> Certificate setup scenario 1`. Other certificate scenarios lower down in the `zowe.yaml` file are commented out.
```

Comment:
Do we have a link to these scenarios? When we do, we should add these scenario links.

Reply:
Ok. Discussions with API ML devs agree that these scenarios should also be in Zowe docs. I can address this.

Reply:
> Ok. Discussions with API ML devs agree that these scenarios should also be in Zowe docs. I can address this.

I have added the links to each scenario.